.vscode-format/cmake-format.py

```diff
@@ -0,0 +1,240 @@
+# ----------------------------------
```

Comment:
Curiosity, these standards, come from OSRF?

Reply:
No. I defined this style in a way it looks similar to some core ROS2 packages. At first, I enjoyed its effect. However, I'm not liking it anymore. I get some weird effects sometimes... I can remove it if you agree

Reply:
I do not oppose, if it make easier to follow ROS2 standard, I would say keep it